final_project/dags/sql/enrich_transfer_from_silver_to_gold.sql

```diff
@@ -0,0 +1,19 @@
+DELETE FROM `gold.user_profiles_enriched` WHERE TRUE;
```

Comment:
You delete all the data here, so users will not have any data for some time. This might lead to issues.